app/dashboard/migrations/0047_auto_20180320_1718.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 2.0.3 on 2018-03-21 00:18
```

Comment:
You'll need to delete and recreate your added migrations for `dashboard` in this branch.

Reply:
was this something that @owocki did above?